tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -545,11 +544,11 @@ where
     ) -> Result<Self> {
         const TPU_LEADER_SERVICE_CREATION_TIMEOUT: u64 = 20;
         let exit = Arc::new(AtomicBool::new(false));
-        let leader_tpu_service = Self::try_create_leader_tpu_service(
+        let leader_tpu_service = Self::create_leader_tpu_service(
             rpc_client.clone(),
             websocket_url,
             exit.clone(),
-            TPU_LEADER_SERVICE_CREATION_TIMEOUT,
+            Duration::from_secs(TPU_LEADER_SERVICE_CREATION_TIMEOUT),
```

Comment:
It is better to use `Duration` over `u64` everywhere.

```suggestion
        const TPU_LEADER_SERVICE_CREATION_TIMEOUT: Duration = Duration::from_secs(20);
        let exit = Arc::new(AtomicBool::new(false));
        let leader_tpu_service = Self::create_leader_tpu_service(
            rpc_client.clone(),
            websocket_url,
            exit.clone(),
            TPU_LEADER_SERVICE_CREATION_TIMEOUT,
```

Notice that when you are using `Duration` it is clear, at the location where `TPU_LEADER_SERVICE_CREATION_TIMEOUT` is defined, what `20` means.  Is it seconds, minutes, milliseconds?

Reply:
very helpful. thank you